tests/unit/views/ont/ONTRuns.spec.js

```diff
@@ -33,7 +33,9 @@ describe('ONTRuns.vue', () => {
     })
 
     it('orders the data by created_at desc', async () => {
-      expect(wrapper.find('tbody').findAll('tr').at(0).text().includes('ONTRUN-2')).toBeTruthy()
+      wrapper.vm.$nextTick(() =>
+        expect(wrapper.find('tbody').findAll('tr').at(0).text().includes('ONTRUN-2')).toBeTruthy(),
```

Comment:
You have to be careful about performing assertions inside the callback function.  It can be that the assertion fails, but the callback consumes the failure.  This is because the callback happens after the test method has already finished running.  Or at least that's what I've seen in the past.  It's better to use

```
await wrapper.vm.$nextTick()
```

and then perform your assertion.

Reply:
Done